src/domain/session/room/timeline/tiles/BaseMessageTile.js

```diff
@@ -49,6 +49,10 @@ export class BaseMessageTile extends SimpleTile {
         return `https://matrix.to/#/${encodeURIComponent(this.sender)}`;
     }
 
+    get eventId() {
```

Comment:
Actually, might be better if this were in SimpleTile to ensure it's always there. There's a few classes that extent SimpleTile.

Reply:
Good call 👍